.github/workflows/pkg-host-deno.yml

```
@@ -0,0 +1,23 @@
+name: pkg-host-deno
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)